packages/system-upgrade-controller/generated-changes/patch/Chart.yaml.patch

```diff
@@ -12,8 +12,8 @@
  sources:
 -  - "https://github.com/rancher/system-charts/blob/dev-v2.6/charts/rancher-k3s-upgrader"
 +  - "https://github.com/rancher/system-charts/charts/system-upgrade-controller"
- version: 0.3.0
- appVersion: v0.7.5
+ version: 0.3.1
+ appVersion: v0.8.0-rc.1
```

Comment:
Did you change this file manually? If so can you move it to the 1st commit?

Reply:
I didn't change this file. It was changed by `make patch`